repositories.gradle

```diff
@@ -11,6 +11,9 @@ repositories {
   maven {
     url "https://linkedin.jfrog.io/artifactory/open-source/" // GMA, pegasus
   }
+  maven {
+    url "https://jitpack.io" // To resolve org.everit.json package, see this - https://stackoverflow.com/questions/57217224/gradle-compile-failed-could-not-find-com-github-everit-org-json-schemaorg-ever
```

Comment:
The repo url used in build is 
```
maven {
      url "https://repository.mulesoft.org/nexus/content/repositories/public/"
    }
```
not sure if `jitpack` works the same

Reply:
Official document recommends using jitpack.io as URL,  https://github.com/everit-org/json-schema#maven-installation

I also used it for testing on Azure Synapse as spark config value, and it worked.